### DISCUSSION TITLE
How to change component image from traits (type:file)

### DISCUSSION THREAD

**Discussion by zeeshan-algo** [GrapesJS/grapesjs#4951] [Q&A] [answered]
Hi . I want to change component image from trait with type: file . How can i achieve this .

**Comment by artf** [comment, marked as answer]
Currently these are the only available built-in traits: https://grapesjs.com/docs/modules/Traits.html#built-in-trait-types
If you need a file picker, you'd need to [implement your own UI](https://grapesjs.com/docs/modules/Traits.html#define-new-trait-type)
